components/lib/crash/README.md

```diff
@@ -36,6 +36,16 @@ With this minimal setup the crash reporting library will capture "uncaught excep
 
 ⚠️ Note: To avoid conflicting setups do not use any other crash reporting libraries/services independently from this library.
 
+### Recording crash breadcrumbs to supported services
+
+Using the `CrashReporter` instance to record crash breadcrumbs.  These breadcrumbs will then be sent when a crash occurs to aid in debugging.  Breadcrumbs are reported only if the underlying crash reporter service supports it.
+
+```Kotlin
+crashReporter.recordCrashBreadcrumb(
+  CrashBreadcrumb("Settings button clicked", data, "UI", Level.INFO, Type.USER)
+)
+```
+
```

Comment:
Let's maybe mention that this is preferred over using the breadcrumbs mechanism of the service (Sentry) directly since that may not work? :)

Reply:
Sure, I'll do that.  Thanks,